### PR TITLE
Implement cloudinary and url-to-screenshot modules, config variables for cloudinary yet to be implemented

### PR DESCRIPTION
**Pull request by ruben1**

### CODE REVIEW

**Comment by AndrewSouthpaw** [comment]
Failed lint tests; please correct them. LMK when finished and I'll merge. Can check by running `gulp test`

File: server/nightmare/script.js [hunk 1]
```diff
@@ -1,25 +1,32 @@
-var Nightmare = require('nightmare');
-var Promise = require('bluebird');
+//var Nightmare = require('nightmare');
```
Comment (AndrewSouthpaw) [inline server/nightmare/script.js]: Once we confirm this all works, we should change the file path to something not including `nightmare`. Created backlog issue #89.

File: server/nightmare/script.js [hunk 2]
```diff
@@ -1,25 +1,32 @@
...
+    Screenshot(url)
+    .width(width)
+    .height(height)
+    .capture(function(err, img) {
+      if(err) throw err;
+      //fs.writeFile(fileName, img, function(err) {
+        //if(err) throw err
+        //console.log("img saved!!");
+      //})
+      cloudinary.uploader.upload(randomString + ".png", function(result) {
```
Comment (AndrewSouthpaw) [inline server/nightmare/script.js]: I think you need to use `fs.createReadStream`. See here: https://devcenter.heroku.com/articles/cloudinary#using-with-node-js-upload. It's down a little bit.

**Comment by AndrewSouthpaw** [comment]
Actually, forget about the lint tests. I've changed the config to be less stupid. We'll clean up the lint errors once we merge with develop and have the updated configs. However, please still integrate the `fs` thing.

**Comment by ruben1** [comment]
hey, i think it should be ok to merge

**Pull request merged**